index.js

```diff
@@ -1,7 +1,7 @@
 /*
 Usage - convert svelte app to web component
-import component from "svelte-kit"
-new component({component:App,tagname:"hello-world",href="/your/stylesheet.css",attributes:["name"]})
+import Component from "svelte-kit"
```

Comment:
Merge conflict right here, @crisward fixed the typo `svelte-kit` to `svelte-tag`.